src/fastoad_cs25/models/aerodynamics/components/compute_cl_alpha.py

```diff
@@ -16,71 +16,84 @@
+        self.add_input("data:geometry:wing:sweep_25", val=np.nan, units="rad")
         self.add_input("data:geometry:wing:root:chord", val=np.nan, units="m")
         self.add_input("data:geometry:wing:area", val=np.nan, units="m**2")
         self.add_input("data:geometry:wing:tip:thickness_ratio", val=np.nan)
 
-        self.add_output("data:aerodynamics:aircraft:takeoff:CL_alpha", units="1/rad")
-        self.add_output("data:aerodynamics:aircraft:takeoff:CL0_clean")
+        if self.options["low_speed_aero"]:
+            self.add_output("data:aerodynamics:aircraft:low_speed:CL_alpha")
+        else:
+            self.add_output("data:aerodynamics:aircraft:cruise:CL_alpha")
 
     def setup_partials(self):
-        self.declare_partials("*", "*", method="fd")
+        if self.options["low_speed_aero"]:
+            self.declare_partials("data:aerodynamics:aircraft:low_speed:CL_alpha", "*", method="fd")
+        else:
+            self.declare_partials("data:aerodynamics:aircraft:cruise:CL_alpha", "*", method="fd")
 
     def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
         width_max = inputs["data:geometry:fuselage:maximum_width"]
         height_max = inputs["data:geometry:fuselage:maximum_height"]
         span = inputs["data:geometry:wing:span"]
         lambda_wing = inputs["data:geometry:wing:aspect_ratio"]
-        l2_wing = inputs["data:geometry:wing:root:chord"]
-        l4_wing = inputs["data:geometry:wing:tip:chord"]
-        el_ext = inputs["data:geometry:wing:tip:thickness_ratio"]
+        root_chord = inputs["data:geometry:wing:root:chord"]
+        tip_chord = inputs["data:geometry:wing:tip:chord"]
+        tip_thickness_ratio = inputs["data:geometry:wing:tip:thickness_ratio"]
         sweep_25 = inputs["data:geometry:wing:sweep_25"]
         wing_area = inputs["data:geometry:wing:area"]
 
-        mach = 0.2
+        if self.options["low_speed_aero"]:
+            mach = 0.2
+        else:
+            mach = inputs["data:TLAR:cruise_mach"]
 
-        beta = np.sqrt(1 - mach**2)
+        beta = np.sqrt(1.0 - mach**2)
         d_f = np.sqrt(width_max * height_max)
-        fuselage_lift_factor = 1.07 * (1 + d_f / span) ** 2
-        lambda_wing_eff = lambda_wing * (1 + 1.9 * l4_wing * el_ext / span)
-        cl_alpha_wing_low = (
-            2
+        fuselage_lift_factor = 1.07 * (1.0 + d_f / span) ** 2
+        lambda_wing_eff = lambda_wing * (1.0 + 1.9 * tip_chord * tip_thickness_ratio / span)
+        cl_alpha_wing = (
+            2.0
             * np.pi
             * lambda_wing_eff
             / (
-                2
+                2.0
                 + np.sqrt(
-                    4
+                    4.0
                     + lambda_wing_eff**2
                     * beta**2
-                    / 0.95**2
-                    * (1 + (np.tan(sweep_25 / 180.0 * np.pi)) ** 2 / beta**2)
+                    / 0.9025  # =0.95**2
```

Comment:
What caused the modification of this coefficient?

Reply:
If the question is about the 0.9025, it's just to avoid a useless computation. The comment was an attempt to explain that.
If it is about the disappearance of the deg to rad conversion, it is now done by OpenMDAO, since the expected unit for the sweep angle has been changed to radians.